VideoApi/VideoApi.Domain/Room.cs

```diff
@@ -106,5 +114,13 @@ public bool DoesEndpointExist(RoomEndpoint endpoint)
         {
             return RoomEndpoints.Any(x => x.EndpointId == endpoint.EndpointId);
         }
+
+        public void UpdateRoomDetails(string label, string ingestUrl, string pexipNode, string participantUri)
```

Comment:
why are these ones bundled, and not others? maybe we could change the name of the method to represent this group of properties?

Reply:
Probably better to rename to represent what I'm trying to achieve